controllers/ovncontroller_controller.go

```diff
@@ -263,11 +264,45 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 
 	instance.Status.Conditions.MarkTrue(condition.InputReadyCondition, condition.InputReadyMessage)
 
+	dbmap, err := v1beta1.GetDBEndpoints(ctx, helper, instance.Namespace, map[string]string{})
+	if err != nil {
+		// TODO(ihar) is it a valid return while waiting for dbcluster?
+		return ctrl.Result{}, err
+	}
+
+	// TODO(ihar) pull the "SB" string from a const package
+	sbCluster, err := v1beta1.GetDBClusterByType(ctx, helper, instance.Namespace, map[string]string{}, "SB")
+	if err != nil {
+		// TODO(ihar) is it a valid return while waiting for dbcluster?
+		return ctrl.Result{}, err
+	}
+	// TODO(ihar) check that sbCluster is not nil (or change the interface of the ByType function)
+
+	// TODO(ihar) move the code below to a separate helper function
+	// TODO(ihar) introduce a constructor for internal-SB string
+	externalConfig := make(map[string]string)
+	// TODO(ihar) should we fail on missing key? perhaps the interface of GetDbEndpoints should be more stringent
+	val, ok := dbmap["internal-SB"]
```

Comment:
Considering the config map would be used for EDP, we would need to expose dbmap["SB"] instead of dbmap["internal-SB"]